tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    /* Visit https:
+    /* Visit https: */
```

Comment:
Looks like this line is junk that should be removed.

Reply:
Done, see https://github.com/dashhive/crowdnode.js/pull/41